src/migration/ember-addon/steps/create-files-from-blueprint.js

```diff
@@ -1,11 +1,21 @@
 import { readFileSync } from 'node:fs';
-import { join } from 'node:path';
+import { dirname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
 
 import glob from 'glob';
 
 import { createFiles } from '../../../utils/files.js';
 import { processTemplate } from '../../../utils/process-template.js';
 
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
```

Comment:
`__dirname` is unavailable in Node. An alternative came from https://stackoverflow.com/a/50052194.